openquakeplatform_building_class/templates/building-class/building-class.html

```diff
@@ -79,7 +78,7 @@ <h1>{% trans "Building Classification Survey" %}</h1>
 
 {% block extra_script %}
     {{block.super}}
-    {% include 'search_scripts.html' %}
+    {#{% include 'search_scripts.html' %}#}
```

Comment:
remove instead of comment.

Reply:
Fixed with this 828d81e0221f19983e8848ea39ebd72625749c56